node/src/effect/requests.rs

```diff
@@ -370,26 +370,29 @@ pub struct ListForInclusionRequest {
     /// request was made. Block Proposer uses this in order to determine if there might be any
     /// deploys that are neither in `past_deploys`, nor among the finalized deploys it knows of.
     pub(crate) next_finalized: u64,
+    /// Random bit with which to construct the `ProtoBlock` requested.
+    pub(crate) random_bit: bool,
     /// Responder to call with the result.
-    pub(crate) responder: Responder<HashSet<DeployHash>>,
+    pub(crate) responder: Responder<ProtoBlock>,
```

Comment:
We still want/need only to return just the deploy hashes. Then we won't need the random bit here.

Reply:
Handled in a later ticket